### DISCUSSION TITLE
Deactivate Suggestions

### DISCUSSION THREAD

**Discussion by buhtz** [WeblateOrg/weblate#9443] [Q&A] [answered]
I was looking for the option to deactivate the possibility for Suggestions.

I would like every contributor should create an account first before translating something.

But I can't found that option.

**Comment by nijel** [comment]
It is a setting at the component level, see https://docs.weblate.org/en/latest/admin/projects.html#enable-suggestions.

**Reply by buhtz** [reply, marked as answer]
To be more precise (at Weblate 4.17): Component -> Manage -> Settings -> Tranlsation -> "Turn on suggestions"

![image](https://github.com/WeblateOrg/weblate/assets/11861496/ea495512-3d29-4561-a8ba-bd2a6b1ae337)
